loadtest/store/memstore/store.go

```diff
@@ -304,7 +304,7 @@ func (s *MemStore) SetChannelMembers(channelMembers *model.ChannelMembers) error
 		return errors.New("memstore: channelMembers should not be nil")
 	}
 
-	for _, member := range *channelMembers {
+	for idx, member := range *channelMembers {
 		// Initialize the maps as necessary.
```

Comment:
That's one way to fix this. The more idiomatic way is to just do a `member := member` inside the loop. We can ignore the idx.

Reply:
Wouldn't that be an additional allocation? I like to use index cause we end up pointing to the original object.

Reply:
By additional allocation, yes a new variable on the stack. It redeclares a new pointer which allows any part of the code to ues it freely and therefore prevents it getting pointed to the last item of the slice.

It's a standard way to doing this: https://github.com/golang/go/wiki/commonmistakes#using-reference-to-loop-iterator-variable.